qemu/tests/virtio_fs_share_data.py

```diff
@@ -548,6 +569,94 @@ def viofs_svc_stop_start(action, cmd, expect_status):
+                        :param object: file or folder
+                        :param xattr: attribute name
+                        :param get_type: return context type or not
+                        :return: context type
+                        """
+                        format = xattr + "=" + context_pattern
+                        full_pattern = re.compile(r'%s' % format)
+                        if side == "host":
+                            xattr_content = process.system_output(getfattr_cmd %
+                                                                  (xattr, object),
+                                                                  shell=True
+                                                                  ).decode().strip()
+                            result = re.search(full_pattern, xattr_content)
+                        if side == "guest":
+                            xattr_content = session.cmd_output(getfattr_cmd %
+                                                               (xattr, object))
+                            result = re.search(full_pattern, xattr_content)
+                        if not result:
+                            test.fail("Attribute is not correct, the pattern is %s\n"
+                                      " the attribute is %s." % (full_pattern,
+                                                                 xattr_content))
+                        if get_type:
+                            return result.group(1)
+
+                    def check_security_label(file, folder, xattr_name):
+                        """
+                        Make sure file context type is the same with it's folder.
+                        only for shared folder in guest as the attribute name is
+                        remapped to others.
+                        :param file: file
+                        :param folder: folder
+                        :param xattr_name: attribute name
+                        """
+                        context_type_file = check_attribute(file, xattr_name)
+                        context_type_folder = check_attribute(folder, xattr_name)
+                        if not context_type_file == context_type_folder:
+                            test.fail("Context type isn't correct.\n"
+                                      "File context type is %s\n"
+                                      "Shared folder context type is %s"
+                                      % (context_type_file,
+                                         context_type_folder))
+
+                    test.log.info("Security.selinux xattr check with xattr mapping.")
+                    # make sure selinux is enabled on host
+                    se_mode = "enforcing"
+                    error_context.context("Set selinux to %s status on"
+                                          " guest." % se_mode, test.log.info)
+                    se_mode_guest_before = session.cmd_output("getenforce").strip()
```

Comment:
"se_mode_guest_before"  
The assignment of "se_mode_guest_before" is in the presence of "getfattr_cmd". Maybe we should set a default value globally?

Reply:
Make sense, I modified code and pls have a check again.